package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "yaml-plus-json",
 	"displayName": "YAML ❤️ JSON",
 	"description": "Easily convert yaml to json and json to yaml",
-	"version": "1.12.1",
+	"version": "1.12.2",
```

Comment:
We can skip bumping version here I will do it just before publishing updated version to ensure git tags match too, like was done here: https://github.com/hilleer/vscode-yaml-plus-json/commit/bf4cd48c20a74afb1afc98b587fe6af2f3e7fa78